coordinator/src/services/handlers/spine-handler.ts

```diff
@@ -22,20 +22,20 @@ const httpsAgent = new https.Agent({
 export class LiveRequestHandler implements RequestHandler {
   private readonly spineEndpoint: string
   private readonly spinePath: string
-  private readonly ebXMLBuilder: (spineRequest: SpineRequest) => string
+  private readonly ebXMLBuilder: (spineRequest: SpineRequest, xRequestIdHeader: string) => string
```

Comment:
At this point, its not a header any more, its just a plain request ID. Could we rename this to `requestId`?

Reply:
Done